dsl/dsl.c

```diff
@@ -157,11 +163,23 @@ static DSLProcBind pbinds [] = {
 	  .helpstr = "(+ <integer> <integer>) -> <integer>", },
 	{ "-",               builtin_sub,          NULL, DSL_PATTR_CHECK_ARITY, 2,
 	  .helpstr = "(- <integer> <integer>) -> <integer>", },
-	{ "string->regexp",  NULL,                 NULL, 0, 0,
+	{ "concat",   builtin_string_append,NULL, 0, 0,
+	  .helpstr = "(concat any<string> ...) -> <string>; an alias for string-append",
+	  .macro = macro_string_append },
+	{ "string-append",   builtin_string_append,NULL, 0, 0,
+	  .helpstr = "(string-append any<string> ...) -> <string>",
+	  .macro = macro_string_append },
+	{ "string->regexp",  builtin_string2regexp,NULL, 0, 0,
 	  .helpstr = "((string->regexp \"PATTERN\") $target) -> <boolean>; PATTERN must be string literal.",
-	  .macro = string2regex},
+	  .macro = macro_string2regexp },
+	{ "regexp-quote",    builtin_regexp_quote, NULL, DSL_PATTR_CHECK_ARITY, 1,
+	  .helpstr = "(regexp-quote <string>) -> <string>",
+	  .macro = macro_regexp_quote },
 	{ "print",   bulitin_debug_print, NULL, DSL_PATTR_CHECK_ARITY, 1,
 	  .helpstr = "(print OBJ) -> OBJ" },
+	{ "print-code", NULL, 0, 0,
+	  .helpstr = "(print-code CODE) -> COD; unlike `print', this works at compile time.",
```

Comment:
```suggestion
	  .helpstr = "(print-code CODE) -> CODE; unlike `print', this works at compile time.",
```

And maybe it's better to use `OBJ` rather than `CODE`? That's what `print` operator uses.

Reply:
Thank you.

How about `EXPRESSION`?  I think `OBJ` implies something already evaluated. However, `print-code` works at compile-time, I thought using another work was better.
```
(print-expr EXPR) -> EXPR; ...
```

Reply:
Yes, `EXPR` is better. In fact we have some help strings that uses `exp`:

```
eq?: (eq? exp0 exp1) -> <boolean>
```

maybe it's good to unify the term.

Reply:
For runtime evaluation, I think we can use OBJ.
For compile-time action(?), we can use EXPR.